apps/src/code-studio/assets/SpriteUpload.jsx

```diff
@@ -1,330 +1,520 @@
 import React from 'react';
 import color from '@cdo/apps/util/color';
 import {makeEnum} from '@cdo/apps/utils';
+import ValidationStep, {Status} from '../../lib/ui/ValidationStep';
 import {
   getManifest,
   getLevelAnimationsFiles,
-  uploadSpriteToAnimationLibrary,
+  uploadAnimationToAnimationLibrary,
   uploadMetadataToAnimationLibrary
 } from '@cdo/apps/assetManagement/animationLibraryApi';
 
-const SpriteLocation = makeEnum('library', 'level');
-const UploadStatus = makeEnum(
-  'fail',
-  'success',
-  'filenameOverride',
-  'badFilename',
-  'none'
-);
+const EXTENSION_CHECK = 'extensionError';
```

Comment:
Not sure, but I wonder about storing these as an object or enum. I don't think that buys us anything programmaticly, but it migth help organize these values as "Errors". No action needed, but wanted to put a bug in your ear.

Reply:
Okay, looking onward at how this is issued - I think I do prefer this as is. :)